mtp_cashbook/apps/feedback/forms.py

```diff
@@ -0,0 +1,17 @@
+from django.utils.text import slugify
+from mtp_common.auth.api_client import get_connection
+from mtp_common.auth.exceptions import Unauthorized
+from slumber.exceptions import HttpClientError
+from zendesk_tickets.forms import EmailTicketForm
+
+
+class PrisonTicketForm(EmailTicketForm):
+    def submit_ticket(self, request, subject, tags, ticket_template_name, **kwargs):
+        try:
+            client = get_connection(request)
+            prisons = sorted(map(lambda prison: prison['name'], client.prisons.get()['results']))
+            subject += ' (%s)' % ', '.join(prisons)
```

Comment:
we wouldn't need to translate this to welsh; it's a ticket that gets created in zendesk – would be odd to arbitrarily make some tickets in welsh. i still do not agree that the above is any clearer, but since it makes you happy…